addons/website_sale/static/src/js/website_sale_tracking.js

```diff
@@ -60,7 +60,7 @@ $(document).ready(function () {
     }
 
     function track_ga() {
-        website_ga = this._gaw || function(){};
+        website_ga = this._gaw || this.ga || function(){};
```

Comment:
Hello,

_gaw can be removed...

Error has been created by commit 7c647ec8 (forward port from saas-4 to saas-5 reset by error the _gaw from initial commit 8eb97cc0829290d7a1f3c2b1b007e257a5615b4c)

So just replace _gaw by ga should be the fix !

thanks for your pr